### PR TITLE
Added Qgis icon (#1694)

### PR DESCRIPTION
**Pull request by Tilak999**
**Issue:**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
  - Why did you pick the hex value? 
      * Hex value provided in design docs.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -2385,6 +2385,11 @@
             "hex": "3776AB",
             "source": "https://www.python.org/community/logos/"
         },
+        {
+            "title": "Qgis",
+            "hex": "589632",
+            "source": "https://www.qgis.org/"
```
Comment (ericcornelissen) [inline _data/simple-icons.json]: For future reference it is easier of the source is the styleguide :wink: 

```suggestion
            "source": "https://www.qgis.org/en/site/getinvolved/styleguide.html"
```

File: icons/qgis.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+  <title>Qgis icon</title>
+  <path d="M12.879 13.006v3.65l-3.004-3.048v-3.495h3.582l2.852 2.893h-3.43zm10.886 7.606V24h-3.654l-5.73-5.9v-3.55h3.354l6.03 6.062zm-10.828-1.448l3.372 3.371c-1.309.442-2.557.726-4.325.726C5.136 23.26 0 18.243 0 11.565 0 4.92 5.136 0 11.984 0 18.864 0 24 4.952 24 11.565c0 2.12-.523 4.076-1.457 5.759l-3.625-3.725a8.393 8.393 0 00.24-2.005c0-4.291-3.148-7.527-7.1-7.527-3.954 0-7.248 3.236-7.248 7.527s3.33 7.6 7.247 7.6c.548 0 .661.017.88-.03z"/>
+</svg>
```
Comment (ericcornelissen) [inline icons/qgis.svg]: For a variety of reasons we like the SVG to be a single line, can you make sure the entire SVG is on one line? That is :

```svg
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Qgis icon</title><path d="M12.879 13.006v3.65l-3.004-3.048v-3.495h3.582l2.852 2.893h-3.43zm10.886 7.606V24h-3.654l-5.73-5.9v-3.55h3.354l6.03 6.062zm-10.828-1.448l3.372 3.371c-1.309.442-2.557.726-4.325.726C5.136 23.26 0 18.243 0 11.565 0 4.92 5.136 0 11.984 0 18.864 0 24 4.952 24 11.565c0 2.12-.523 4.076-1.457 5.759l-3.625-3.725a8.393 8.393 0 00.24-2.005c0-4.291-3.148-7.527-7.1-7.527-3.954 0-7.248 3.236-7.248 7.527s3.33 7.6 7.247 7.6c.548 0 .661.017.88-.03z"/></svg>
```

File: _data/simple-icons.json (hunk 1 above)
Comment (Tilak999) [reply _data/simple-icons.json]: Thanks for guiding, made the changes in latest commit. :)

File: icons/qgis.svg (hunk 2 above)
Comment (Tilak999) [reply icons/qgis.svg]: Changes done 👍

**Pull request merged**